packages/client/src/views/PrintCertificate/PDFUtils.ts

```diff
@@ -124,9 +124,33 @@ function getPDFTemplateWithSVG(
 ): IPDFTemplate {
   let svgTemplate
   if (declaration.event === Event.Birth) {
-    svgTemplate = offlineResource.templates.certificates!.birth.definition
+    if (
+      offlineResource.templates.certificates!.birth.definition.includes(
+        'data:image/svg+xml;base64'
+      )
+    ) {
+      svgTemplate = atob(
+        offlineResource.templates.certificates!.birth.definition.split(
+          'data:image/svg+xml;base64,'
+        )[1]
+      )
+    } else {
+      svgTemplate = offlineResource.templates.certificates!.birth.definition
+    }
   } else {
-    svgTemplate = offlineResource.templates.certificates!.death.definition
+    if (
+      offlineResource.templates.certificates!.death.definition.includes(
+        'data:image/svg+xml;base64'
+      )
+    ) {
+      svgTemplate = atob(
+        offlineResource.templates.certificates!.death.definition.split(
+          'data:image/svg+xml;base64,'
+        )[1]
+      )
+    } else {
+      svgTemplate = offlineResource.templates.certificates!.death.definition
+    }
   }
```

Comment:
Lets use the `declaration.event` to reduce similar code. something like:
`offlineResource.templates.certificates![declaration.event].definition.includes`

Reply:
> Lets use the `declaration.event` to reduce similar code. something like: `offlineResource.templates.certificates![declaration.event].definition.includes`

I have updated the pr with the demanded changes.